packages/client/hmi-client/src/services/charts.ts

```diff
@@ -749,7 +750,7 @@ export function createInterventionChartMarkers(
 			type: 'text',
 			align: 'left',
 			angle: 90,
-			dx: 5,
+			dx: labelXOffset,
 			dy: -10
```

Comment:
Why only `dx` deserves a constant?

Reply:
It's a param, there's nowhere in the code that changes `dy`